ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts

```diff
@@ -107,7 +107,8 @@ export interface RenewRegistrationIF {
   createDateTime?: string, // Included in a successful response.
   payment?: PaymentIF, // Included in a successful response.
   error?: ErrorIF,
-  authorizationReceived?: boolean // Always true for the UI: API requirement.
+  authorizationReceived?: boolean, // Always true for the UI: API requirement.
+  courtOrderInformation?: CourtOrderIF, // Only populated if RL renewal and all court order elements present.
```

Comment:
does it let you have a trailing comma there?

Reply:
It does, but I removed the comma anyway.